app/jobs/scrape_job.rb

```diff
@@ -0,0 +1,66 @@
+class PostScrapeWorker < ActiveJob::Base
+  queue_as :default
+
+  require 'capybara'
+  require 'capybara/poltergeist'
+
+  include Sidekiq::Worker
+  include Capybara::DSL
+  Capybara.register_driver :poltergeist do |app|
+    Capybara::Poltergeist::Driver.new(app, timeout: 60, js_errors: false)
+  end
+  Capybara.default_driver = :poltergeist

```

Comment:
The capybara config here on lines 9 to 12 (**not** the include on line 8) might be better in an initializer in `config/initializers`. Consider moving this config out of the 2 jobs it is currently in and put it in a single initializer.